conans/test/functional/layout/test_in_cache.py

```diff
@@ -43,6 +43,75 @@ def package_info(self):
     return conan_file
 
 
+def test_create_test_package_no_layout(conanfile):
+    """The test package using the new generators work (having the generated files in the build
+    folder)"""
+    client = TestClient()
+    conanfile_test = textwrap.dedent("""
+        import os
+
+        from conans import ConanFile, tools
+        from conan.tools.cmake import CMakeToolchain, CMake, CMakeDeps
+
+        class HelloTestConan(ConanFile):
+            settings = "os", "compiler", "build_type", "arch"
+
+            def generate(self):
+                deps = CMakeDeps(self)
+                deps.generate()
+                tc = CMakeToolchain(self)
+                tc.generate()
+
```

Comment:
```suggestion

        class HelloTestConan(ConanFile):
            settings = "os", "compiler", "build_type", "arch"
            generators = "CMakeDeps", "CMakeToolchain"
```

Just trying to make tests more compact